app/assets/javascripts/project.js

```diff
@@ -64,3 +64,30 @@ window.addEventListener("load", () => {
     });
   });
 });
+
+function debounce(func, timeout = 300) {                                                
+  let timer;
+  return (...args) => {
+    clearTimeout(timer);
+    timer = setTimeout(() => { func.apply(this, args); }, timeout);
+  };
+};
+
+const filterStories = debounce(() => {
```

Comment:
do we really need to debounce the filtering? since we are not doing backend requests there's no problem on filtering right after the input changes I guess, isn't this adding an artificial delay?

Reply:
You're right. Removed the debouncing.